news/bugfix-3407.md

```diff
@@ -0,0 +1 @@
+systemd-journal: add namespace to the persist name
```

Comment:
I don't want to nitpick with news file reviews, but can we emphasize here what you wrote in PR description, please?

> ....  allow multiple systemd-journal sources at the same time. .....  two journal sources are configured to read from two different namespaces they can work side by side.

Reply:
The current version pushed forward to support multiple `systemd-journal()` source, yet not lifted the restriction. From user point of view the one instance limitation still stands. As such, it would not help to mention in the news file.

Reply:
Yes, you are right, I forgot the `journal_reader_initialized` global variable restriction...

https://github.com/syslog-ng/syslog-ng/blob/master/modules/systemd-journal/journal-reader.c#L630